heritability/import_heritability.py

```diff
@@ -0,0 +1,54 @@
+__author__ = 'Rahul Gupta'
+
+import hail as hl
+
+
+def get_h2_flat_file():
+    return 'gs://ukb-diverse-pops/rg-h2-tables/h2_estimates_all_flat.tsv'
+
+
+def get_h2_ht():
+    return 'gs://ukb-diverse-pops/rg-h2-tables/ht/h2_estimates_all.ht'
+
+
+def import_h2_flat_file(save_to_ht, overwrite):
+    ht = hl.import_table(get_h2_flat_file(), 
+                         delimiter='\t', 
+                         impute=True, 
+                         key=['trait_type','phenocode','pheno_sex','coding','modifier'])
+    ht = ht.rename({'ancestry':'pop'})
+    ht = ht.drop('phenotype_id')
+    # solution from Zulip to munge flat file columns into nested structs
+    d = {}
+    row = ht.row
+    for name in row:
+        if name not in ht.key:
+            def recur(dict_ref, split_name):
```

Comment:
Can you pull these functions outside the for loop? Now that I'm saying it out loud, I don't know why we do that, but that is our typical convention 🤔